### DISCUSSION TITLE
blank page after trying to create new metamask wallet at connect wallet modal

### DISCUSSION THREAD

**Discussion by Mohammed-Mamoun98** [rainbow-me/rainbowkit#2157] [Feedback]
<img width="435" alt="Connect a Wallet" src="https://github.com/user-attachments/assets/5d4c6bf1-e754-49ff-9a24-79e439f5bfae">
![MetaMask](https://github.com/user-attachments/assets/b8452813-fa9d-4472-8e5d-8d40011bdf29)

**Comment by Mohammed-Mamoun98** [comment]
I find the issue also on ur main website

**Comment by DanielSinclair** [comment]
Thanks for reporting. I am able to reproduce, so I opened an issue to track: https://github.com/rainbow-me/rainbowkit/issues/2158
